zuul.d/projects.yaml

```diff
@@ -391,3 +391,10 @@
     3pci-check:
       jobs:
         - noop
+
+- project:
+    name: github.com/ansible/workshops
+    default-branch: devel
```

Comment:
should this be master? if so, can be removed.

Reply:
`devel` is the right value here.